### DISCUSSION TITLE
Compromised GitHub Account - Need Assistance Regaining Access

### DISCUSSION THREAD

**Discussion by Sebastian-vas** [community/community#127332] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Five days ago, my GitHub account was hacked. Despite having two-factor authentication (2FA) enabled, the hackers were able to bypass it and change my email address. As a result, I have completely lost access to my account.

Upon realizing the situation, which was approximately 3-5 hours after the hack, I immediately created a support ticket. However, to date, I have not received any response from you.

Through this means, I would like to inquire about the average response time for support tickets, as I urgently need to access the repositories hosted in my account. Due to this incident, I have been unable to make progress on my project and I am increasingly frustrated by the lack of response from the support team.

I kindly request that my case be prioritized and that you provide me with an update on the status of my support ticket as soon as possible. I would also appreciate any guidance you can offer me to regain access to my account.

I appreciate your time and attention to this matter.

**Comment by 0xshubhs** [comment]
#127332 
Hello 13Pingg,

I'm sorry to hear about your compromised GitHub account. Losing access to your account, especially with two-factor authentication enabled, can be quite concerning.

Unfortunately, GitHub's response time to support tickets can vary, and it's frustrating not to receive a prompt update in such situations.

Here are a few steps you can take to expedite the process and regain access to your account:

1. **Follow Up on Your Support Ticket:**
   - Double-check that you've provided all necessary information in your support ticket.
   - If you haven't received a response in a timely manner, consider updating your ticket with any additional details or following up directly through GitHub's support channels.

2. **Review Your Email and Account Security:**
   - Ensure that your email account associated with GitHub hasn't been compromised as well.
   - Review your email's security settings and change your email account password if necessary.

3. **Stay Proactive:**
   - Continue to check your email and GitHub notifications for any updates from the GitHub support team.
   - If you haven't already, enable two-factor authentication (2FA) on your email account if it isn't already.

4. **Alternative Contact Methods:**
   - GitHub has a support page where you can contact their support team directly. You can also try reaching out to them through their official social media channels for additional support.

I hope this helps and that you regain access to your account soon. If you have any other questions, feel free to ask.

**Comment by Sebastian-vas** [comment]
Thank you for taking the time to respond to my ticket. I appreciate your willingness to help me resolve this issue.

I understand that GitHub's support ticket response times can vary, but I'm concerned about the delay in addressing my ticket. I submitted a ticket five days ago regarding a compromised account, and I haven't received any updates or resolution yet. I've reviewed the support page, and it seems that there is no real-time support available.

I'm particularly concerned about the fact that the hacker was able to gain access to my account despite having two-factor authentication enabled. I've already changed my Gmail password as a precautionary measure.

To expedite the process, I've provided additional information in my ticket and will continue to monitor my email and GitHub notifications for any updates. However, I would greatly appreciate an estimated timeframe for when I can expect a response or resolution. My work depends on my GitHub account, and I have a project on hold due to this issue.

In addition to the account access issue, I'm also concerned about the possibility that the hacker may have deleted my repositories. Is there any way for the GitHub staff to assist me in recovering these repositories?

I understand that you may not have direct access to the information I need, but I would be grateful if you could escalate my issue to the appropriate team or provide me with further guidance on how to proceed.

Thank you for your time and consideration. I look forward to a prompt resolution.

**Reply by LiteBrite82** [reply, marked as answer]
Hi @13Pingg 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

Also, if you have not already, check out this documentation on [2FA account recovery](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials)

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Comment by git-param** [comment]
same here...my account got hacked at 1 am which is best time for hackers....they reset my password and deleted my account and github support team is not able to do anything...i lost all my stats....what to do now

**Reply by msaimnaveed2005** [reply]
bro mine account is hacked too and the hacker removed the main email from my account

**Reply by blue-lands** [reply]
same here

**Reply by msaimnaveed2005** [reply]
hey brother i regained my account
If you have backup email still attached and you can login with it than you can change your backup email to a non backup email and than you can remove the hackers email and enable two factor authentication and change your password. ALso remove any apis which you dont know and are attached to your account

**Comment by msaimnaveed2005** [comment]
Hi, yes, my account was compromised as well.
I also contacted GitHub Support, but unfortunately, I didn't receive a response. I tried asking the community for help too, but I couldn't find a solution at the time.
After about two weeks, I checked my account again and remembered that I had previously added a backup email address. I was able to regain access using that backup email.
Once I got back in, I went to Settings → Emails, verified my backup email, and was then able to remove the attacker's primary email address. I made my secure, 2FA-protected email address the new primary email and removed any repositories that had been published by the attacker.
After recovering the account, I also strengthened its security by enabling 2FA and reviewing my account settings and repositories.
So, in my case, GitHub Support unfortunately didn't respond, and I was able to recover the account myself through the backup email. I hope this helps, and I hope you're able to regain access to your account soon.
________________________________
